src/image.tsx

```diff
@@ -60,16 +61,17 @@ type DeprecatedImageProps = {
     aspectRatio?: AspectRatio;
     /** defaults to empty string */
     alt?: string;
-    children?: void;
     dataAttributes?: DataAttributes;
+    children?: void;
+    borderRadius?: boolean;
 };
 
 const Image = React.forwardRef<HTMLImageElement, ImageProps | DeprecatedImageProps>(
-    ({aspectRatio = '1:1', alt = '', dataAttributes, ...props}, ref) => {
-        const noBorderRadius = useDisableBorderRadius();
+    ({aspectRatio = '1:1', alt = '', dataAttributes, borderRadius = false, ...props}, ref) => {
```

Comment:
this is a breaking change, until now, Images had border radius by default.
How are we going to manage this? Should we release a major?